.github/workflows/opensearch-observability-test-and-build-workflow.yml

```diff
@@ -46,5 +46,72 @@ jobs:
       - name: Upload Artifacts
         uses: actions/upload-artifact@v1
         with:
-          name: opensearch-observability
+          name: opensearch-observability-ubuntu
           path: opensearch-observability-builds
+
+  windows-build:
+    strategy:
+      matrix:
+        java:
+          - 11
+          - 17
+
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
```

Comment:
Same thing here, I am skipping the backwards compatibility tests as they were failing with windows not recognizing: 
`-Dtests.security.manager=false`
Again, not sure if its okay to skip some of the testing phases on windows

Reply:
Can you paste the error logs here? We didn't change code for the platform specifically in theory it should work.

Reply:
For MacOS Im getting the following: 
`Could not HEAD 'https://artifacts.opensearch.org/releases/core/opensearch/1.2.0/opensearch-min-1.2.0-darwin-x64.tar.gz'. Received status code 403 from server: Forbidden`

For Windows, Im getting the following:
`Task '.security.manager=false' not found in root project 'opensearch-observability'.`

See: https://github.com/opensearch-project/observability/actions/runs/3236985158/jobs/5303492955

Reply:
Would you be able to help me debug these issues? @ps48

Reply:
BWC not working should be ok, you can add the lines and comment them out. because this is the initial release on windows, i don't think we have anything to test upgrade against on.

Reply:
@derek-ho For Mac-OS, the opensearch-min 1.2.0 artifact is not found as older versions are not built. Need to look into how windows flags are used. For now, As Josh mentioned let's comment out these lines and come back later once things are stable.

Reply:
Done